plugins/quay/src/handlers.ts

```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { rest } from 'msw';
+
+const LOCAL_ADDR = 'https://localhost:5050/quay/';
+
+export const handlers = [
```

Comment:
Since it looks like these handlers are only being used for testing purposes by `plugins/quay/src/index.ts`, you can just move this definition to `index.test.ts` and delete this file. 

In general, I would expect a file named `src/handlers.ts` to be exposing API route handlers that are consumed by a server somewhere. So if you do plan on consuming these routes from multiple test files, you would want to name the file something like `test-handlers.ts`. if they are meant to be consumed by multiple files.